src/net/tcp_stream.rs

```diff
@@ -105,6 +105,9 @@ impl TcpStream {
     /// small amount of data. When not set, data is buffered until there is a
     /// sufficient amount to send out, thereby avoiding the frequent sending of
     /// small packets.
+    ///
+    /// Note On Windows: Make sure the stream is connected before calling this method.
```

Comment:
Can you changes this to:

```rust
    /// # Notes
    /// 
    /// On Windows make sure the stream is connected before calling this method,
    /// by receiving an (writable) event. Trying to set `nodelay` on an
    /// unconnected `TcpStream` is undefined behaviour.
```

Same for the others.

Reply:
Done, reworded.

Reply:
You missed the Notes header.

Reply:
Sorry for that, I've added it.